### PR TITLE
translating translating policies and telemetry config into Chinese

### PR DESCRIPTION
**Pull request by rootsongjc**
I have reviewed it before, there's no change since the last commit in istio.dochzh.io
@gyliu513 @fleeto please review

### CODE REVIEW

File: content_zh/docs/concepts/policies-and-telemetry/config/index.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+```
+
+Mixer 是实现策略和遥测功能的 Istio 组件。Mixer 本质上是一个属性处理机器。Envoy sidecar 为每个请求调用 Mixer，为 Mixer 提供一组描述请求和请求周围环境的属性。基于它的配置和给定的特定属性集，Mixer 会调用各种基础设施后端。
+
+{{< image width="60%" ratio="42.60%"
+    link="/docs/concepts/policies-and-telemetry/config/machine.svg"
+    caption="属性机"
+    >}}
+
+### Attribute vocabulary
```
Comment (gyliu513) [inline content_zh/docs/concepts/policies-and-telemetry/config/index.md]: kill this

**Comment by rootsongjc** [comment]
@gyliu513 OK, deleted.

**Comment by rootsongjc** [comment]
@sdake  I fix a broken error about mysql shell.

**Comment by gyliu513** [comment]
@rootsongjc can you put the change `content/blog/2018/egress-tcp/index.md` to a new PR? This can make sure we can merge the Chinese version first.

**Comment by rootsongjc** [comment]
OK. The reason I update the English content links is that to make sure the CI go successfully. I will revert and make a new PR for English content separately.

**Pull request closed without merging**